src/css/components/status_pill.scss

```diff
@@ -0,0 +1,13 @@
+
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.vertical-pill {
```

Comment:
The name of the CSS class and the file name should probably match. Can we call `vertical-pill` `status-pill`, or other way around?